_opensearch/point-in-time-api.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: default
+title: Point in Time API
+nav_order: 58
+has_children: false
+parent: Point in Time
+---
+
+# Point in Time API
+
+Use the [Point in Time (PIT)]({{site.url}}{{site.baseurl}}/opensearch/point-in-time/) API to manage Points in Time. 
+
+---
+
+#### Table of contents
+- TOC
+{:toc}
+
+---
+
+## Create a PIT
```

Comment:
Can we also add info about these cluster settings.

- We limit the maximum keep-alive of a PIT using a cluster level setting  `point_in_time.max_keep_alive` (defaults to 24h . Users can change it as required.])
-  We limit the number of open PIT contexts on each node using a node level setting  `search.max_open_pit_context` (Defaults to 300.)